### PR TITLE
updated home styling

### PR DESCRIPTION
**Pull request by shivambalwani**
fixed issue #3:
 changed inline styling in home page to external styling

### CODE REVIEW

**Comment by pranshuchittora** [comment]
Thanks for the PR, but it's not recommended to use direct css due to name collisions.
Pls use the CSS module for the styling

**Comment by pranshuchittora** [comment]
Ref -> https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/

**Comment by shivambalwani** [comment]
> Thanks for the PR, but it's not recommended to use direct css due to name collisions.
> Pls use the CSS module for the styling

No problem sir! changing it

**Comment by shivambalwani** [comment]
sir I have added css modules stylesheet in this pull request. Should i create a new branch for new pull request?

File: src/pages/home/home.module.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+}
+.explore_now{
+    width: 100%;
+     text-align: center;
+      margin-bottom: 2rem;
+}
+.divider{
+    margin: 3vh 0;
+}
+.grid{
```
Comment (pranshuchittora) [inline src/pages/home/home.module.css:17]: Can you name them more specifically

File: src/pages/home/home.module.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    justify-content: space-evenly;
+    
+}
+
+    
+.paper{
```
Comment (pranshuchittora) [inline src/pages/home/home.module.css:23]: This one too

**Comment by pranshuchittora** [comment]
All class names seems to generic, it would be great if they are more specific

**Review by pranshuchittora — CHANGES_REQUESTED** [review]
Fix class names

**Comment by shivambalwani** [comment]
> Fix class names

@pranshuchittora i have fixed the style naming, but the checks are failing. I don't know why?

**Comment by pranshuchittora** [comment]
Pls pull the new changes and resolve the conflicts.

**Pull request closed without merging**